Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -133,11 +133,7 @@ public extension Attribute where Context == HTML.LinkContext {
     /// - parameter language: The language to assign.
     /// - parameter country: The country of the language to assign.
     static func hreflang(_ language: Language, _ country: Country? = nil) -> Attribute {
-        if let country = country {
-            return Attribute(name: "hreflang", value: "\(language.rawValue)-\(country.rawValue)")
-        } else {
-            return Attribute(name: "hreflang", value: language.rawValue)
-        }
+        Attribute(name: "hreflang", value: country != nil ? "\(language.rawValue)-\(country?.rawValue ?? "")": language.rawValue)
```

Comment:
Personally I prefer this variant over the former it replaces.  But the former is easier to quickly read I think.

Reply:
Yeah, with so much optional unwrapping and string interpolation going on within a single long line, the code becomes really messy IMO.

Reply:
This code has been removed, but I do have a question for you about it.  In this case country != nil tests that country is initialized and contains a value, what is your preference with "!"?  It would seem 100% safe to force unwrap over chaining, so that no default value is necessary.  I must admit I am in the habit of of never force unwrapping mostly because swiftlint complains, and it seems like it may create an edge case.  What rules to this do you apply when coding?